samples/sqlservices/src/apiWrapper.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+import * as vscode from 'vscode';
+import * as data from 'sqlops';
+
+/**
+ * Wrapper class to act as a facade over VSCode and Data APIs and allow us to test / mock callbacks into
+ * this API from our code
+ *
+ * @export
+ * @class ApiWrapper
+ */
+export class ApiWrapper {
+	// Data APIs
+
+	public registerWebviewProvider(widgetId: string, handler: (webview: data.DashboardWebview) => void): void {
+		return data.dashboard.registerWebviewProvider(widgetId, handler);
+	}
+
+
+	public registerControlHostProvider(widgetId: string, handler: (webview: data.DashboardWebview) => void): void {
+		return data.dashboard.registerWebviewProvider(widgetId, handler);
```

Comment:
should this call registerControlHostProvider?

Reply:
Removing this as it's not really needed / used for this sample